st2api/st2api/wsgi.py

```diff
@@ -0,0 +1,42 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pecan import load_app
+from oslo_config import cfg
+
+from st2api import config  # noqa
+from st2common import log as logging
+from st2common.models import db
+
+
+cfg.CONF(args=['--config-file', '/etc/st2/st2.conf'])
```

Comment:
So this will only work with the default config path, right?

Maybe we should document that or it's not an issue right now for AIO installer?